src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ServiceHandlerProps.scala

```diff
@@ -0,0 +1,10 @@
+
+package org.broadinstitute.dsde.agora.server.webservice
+
+import akka.actor.Props
+
+trait ServiceHandlerProps {
```

Comment:
Initial reason was so it could be overriden. In apollo, by test-vs-prod Props. Here, we were thinking different endpoint props. I'd say fold it in to AgoraService